nav2_experimental_rl/random_crawl_train.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from turtlebot3_env import TurtlebotEnv
+from dqn import DQN
+
+import random
+import numpy as np
+import tensorflow as tf
+
+import rclpy
+from rclpy.node import Node
+from time import sleep
+import parameters
+
+def trainModel(env, action_size):
+    state = env.reset()
+    observation_space = len(state)
+    agent = DQN(observation_space, action_size)
+    target_model_update_counter = 0
+    agent.step = 0
+    for _ in range(parameters.EPISODES):
+        print("Episode number: " + str(_))
+        state = env.reset()
+        observation_size = len(state)
+        state = np.reshape(state, [1, observation_size])
+        done = False
+        while not done:
+            agent.step += 1
+            target_model_update_counter += 1
+            if target_model_update_counter%parameters.TARGET_MODEL_UPDATE_STEP == 0:
```

Comment:
We're switching from 4 spaces a line to 2 in the inner if conditions in a few of these files.

Reply:
Fixed.